.circleci/config.yml

```diff
@@ -44,7 +44,8 @@ jobs:
           cd ${CIRCLE_WORKING_DIRECTORY}
           # Install Python dependencies
           pip install --upgrade pip
-          pip install -r requirements.txt
+          pip install toml && python -c 'import toml; c = toml.load("pyproject.toml")
+          print("\n".join(c["project"]["dependencies"]))' | pip install -r /dev/stdin
```

Comment:
I wanted to ask - why not simply do `pip install pdm && pdm install`  instead of parsing the toml file and piping it to pip?

Reply:
Let me know if you have any thoughts! I haven't used CircleCI much to be honest - so it could just be my error 😅

Reply:
Thanks, I'll test with `pdm export`.